tests/ValueParsers/IsoTimestampParserTest.php

```diff
@@ -151,18 +157,29 @@ public function validInputProvider() {
 			'-0000000000000001-01-01T00:00:00Z (Gregorian)' => array(
 				'-0000000000000001-01-01T00:00:00Z',
 				TimeValue::PRECISION_DAY,
+				$gregorian
+			),
+			'+00000002001-01-01T00:00:00Z (Gregorian)' => array(
+				'+0000000000002001-01-01T00:00:00Z',
+				TimeValue::PRECISION_DAY,
+				$gregorian,
+				$julianOpts // overridden by explicit calendar in input string
 			),
-			'-00000000001-01-01T00:00:00Z (Gregorian)' => array(
```

Comment:
Seems like I actually duplicated that test case, the -00000000001 was added below. But I changed it back to make the diff look cleaner.